generators/entity-client/templates/angular/src/main/webapp/app/entities/update/entity-form.service.ts.ejs

```diff
@@ -0,0 +1,214 @@
+<%#
```

Comment:
dumb question: could it be called entity.form.ts?

Reply:
In my opinion makes more sense to call it entity-form.